model/src/cmd/ignmodel.yaml.in

```diff
@@ -0,0 +1,8 @@
+--- # Subcommands available inside ignition-model.
+format: 1.0.0
+library_name: ignition-model
```

Comment:
This can be confusing, because there isn't a library called `ignition-model`.

If we move this into `src/cmd` like I suggested above, I think the `model` command can be listed into `gazebo.yaml.in` as part of the `ignition-gazebo-ign` library.

Reply:
This is something I didn't address atm because it will be depending on the final file structure. I explained it a bit [here](https://github.com/ignitionrobotics/ign-gazebo/pull/893#discussion_r673146560)

Reply:
See #928